slasher/db/schema.go

```diff
@@ -0,0 +1,45 @@
+package db
+
+import (
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+)
+
+// The Schema will define how to store and retrieve data from the db.
```

Comment:
This comment is not relevant, please update

Reply:
updated